packages/api-explorer/src/scenes/TagScene/TagScene.tsx

```diff
@@ -52,7 +52,7 @@ export const TagScene: FC<TagSceneProps> = ({ api }) => {
 
   return (
```

Comment:
Not part of this PR, but I looked at this full file.  On line 45, you filter for the tag, and then take the 0th element.  The more semantic method to use is `find` which will return the your the first object it finds that matches condition.

Reply:
This has been addressed in #255.